Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN apt-get update -y && apt-get install -yy \
       fluent-plugin-elasticsearch:1.4.0 \
       fluent-plugin-record-modifier:0.4.1 \
       fluent-plugin-multi-format-parser:0.0.2 \
-      fluent-plugin-kinesis-aggregation:0.2.2 \
+      fluent-plugin-kinesis-aggregation:0.3.0 \
       fluent-plugin-concat:0.4.0 \
       fluent-plugin-parser:0.6.1 \
```

Comment:
from https://github.com/tagomoris/fluent-plugin-parser:

> NOTE: This plugin is outdated for Fluentd v0.14 (Fluentd v0.14 has 'parser' filter plugin)

I know the above because I was the one to add the plugin-parser to this Docker image. My concern is that there is going to be a lot of testing done for a change for Fluentd version as who know if any other plugins are affected or they work in non backwards compatible ways.